### PR TITLE
Add/setting enable location listening

### PR DESCRIPTION
**Pull request by brennan**
Location listening is enabled by default, so we should give users the option to disable it on initialization via a Segment setting.

### CODE REVIEW

File: src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java [hunk 1]
```diff
@@ -92,6 +92,11 @@ public AmplitudeClient get() {
     boolean trackSessionEvents = settings.getBoolean("trackSessionEvents", false);
     amplitude.trackSessionEvents(trackSessionEvents);
     logger.verbose("AmplitudeClient.getInstance().trackSessionEvents(%s);", trackSessionEvents);
+
+    boolean enableLocationListening = settings.getBoolean("enableLocationListening", true);
```
Comment (ladanazita) [inline src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java]: Does this default to `true`?

Comment (f2prateek) [reply src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java]: We should set the default to whatever is in metadata.

Comment (brennan) [reply src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java]: For context, Amplitude defaults this setting to true, so I think we probably should as well to emulate native behavior.

Comment (f2prateek) [reply src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java]: Makes sense. The default in metadata should be the default behaviour of their SDK, and should be the same value here as well.

**Pull request merged**